app/1.0/docs/devguide/local-dom.md

```diff
@@ -18,6 +18,25 @@ when manipulating DOM from JavaScript. This interface covers most of the
 common DOM methods and properties, and is compatible with both shady DOM and
 native shadow DOM.
 
+```html
+<dom-module>
+  <template>
+  
+    <!-- 
+      Local DOM. Possible implementations : 
+        - Shadow DOM (browser implementation)
+        - Shady DOM (Polymer implementation)
+    -->
+    
+    <content>
+      <!-- 
+        Light DOM
+      -->
+    </content>
+    
+  </template>
+</dom-module>
+```
```

Comment:
Something seems not quite right about this. I will try to explain my thoughts.

I read the the comments in the `<template>` section as instructing me to insert my own code here, and that my code must be either: 

* A "Shadow DOM (browser implementation)", or
* A "Shady DOM (Polymer implementation)"

I would not know what either of these things are, or how to write them. 

Thus I find this suggested addition to be more confusing rather than clarifying.

But I will ask @arthurevans to comment here and will trust his opinion on the matter.